python/doc/examples/reliability_sensitivity/estimate_probability_form.ipynb

```diff
@@ -6563,7 +6580,7 @@
    "name": "python",
```

Comment:
At some point in the notebook,
"# create the event we want to estimate the probability"
should be replaced bu
"# create the event whose probability we want to estimate"

Reply:
Ok. Fixed.